source/query-bar.txt

```diff
@@ -146,7 +146,7 @@ maximum number of documents to return. To specify the limit, click
 Run the Query
 ~~~~~~~~~~~~~
 
-Click the :guilabel:`Apply` button to run the query and view the
+Click the :guilabel:`Find` button to run the query and view the
```

Comment:
Note that these two actions simultaneously - "As you type ... " - so, I'm concerned that breaking this short sentence in two offers more danger to comprehension than potential value.